src/context/CardContext/index.js

```diff
@@ -1,67 +1,26 @@
-import React, { createContext, useState } from "react";
+import React, { createContext, useState, useEffect } from "react";
 import { v4 as uuidv4 } from "uuid";
+import axios from "axios";
 
 export const CardContext = createContext();
 
 const CardContextProvider = (props) => {
-  const [cards, setCards] = useState([
-    {
-      id: "1",
-      head: "Card 1",
-      body: "Some text on card 1",
-      isChecked: false,
-      isEditMode: false,
-    },
-    {
-      id: "2",
-      head: "Card 2",
-      body: "Some text on card 2",
-      isChecked: false,
-      isEditMode: false,
-    },
-    {
-      id: "3",
-      head: "Card 3",
-      body: "Some text on card 3",
-      isChecked: false,
-      isEditMode: false,
-    },
-    {
-      id: "4",
-      head: "Card 4",
-      body: "Some text on card 4",
-      isChecked: false,
-      isEditMode: false,
-    },
-    {
-      id: "5",
-      head: "Card 5",
-      body: "Some text on card 5",
-      isChecked: false,
-      isEditMode: false,
-    },
-    {
-      id: "6",
-      head: "Card 6",
-      body: "Some text on card 6",
-      isChecked: false,
-      isEditMode: false,
-    },
-    {
-      id: "7",
-      head: "Card 7",
-      body: "Some text on card 7",
-      isChecked: false,
-      isEditMode: false,
-    },
-    {
-        id: "8",
-        head: "Card 8",
-        body: "Some text on card 8",
-        isChecked: false,
-        isEditMode: false,
-      },
-  ]);
+  const [cards, setCards] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get(
+        "https://raw.githubusercontent.com/BrunnerLivio/PokemonDataGraber/master/output.json"
+      )
+      .then((res) => {
+        let cards = res.data.slice(0, 15);
+        cards = cards.map((card) => {
+          return { ...card, isChecked: false, isEditMode: false };
+        });
+        setCards(cards);
+      })
```

Comment:
It's good to retain original data properties names Name, About, etc...
However, because of this retainment you had to put multiple changes to several components that could potentially ruin another logic. This problem could have been avoided if you've just re-assigned props during the load.
```
... 
return {
  id: card.Number,
  head: card.Name,
  body: card.About, 
  ...
}
...
```
Also, these could be simplified and re-written with out declaring new variable:
```
setCards(res.data.slice(0,15).map((card) => {
   ...
});
```